code/game/mecha/working/ripley.dm

```diff
@@ -16,6 +16,7 @@
 	var/list/cargo = new
 	var/cargo_capacity = 15
 	var/hides = 0
+	var/plates = 0
```

Comment:
Worth adding autodoc comments to both hides and plates vars.

Reply:
done